ios/App/Shared/player/AudioPlayer.swift

```diff
@@ -300,9 +302,12 @@ class AudioPlayer: NSObject {
         // Start playback, with a seek, for as smooth a scrub bar start as possible
         let currentTrackStartOffset = session.audioTracks[self.currentTrackIndex].startOffset ?? 0.0
         let seekTime = currentTime - currentTrackStartOffset
-        self.audioPlayer.seek(to: CMTime(seconds: seekTime, preferredTimescale: 1000), toleranceBefore: .zero, toleranceAfter: .zero) { [weak self] completed in
-            guard completed else { return }
-            self?.resumePlayback()
+        
+        DispatchQueue.runOnMainQueue {
```

Comment:
Actions that modify the `audioPlayer` must be on the main queue. This was the case 99% of the time before, but by adding this code, we are being explicit it's happening.